catalyst_voices_packages/catalyst_cardano_serialization/lib/src/types.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:cbor/cbor.dart';
+
+/// Specifies an amount of ADA in terms of lovelace.
+typedef Coin = int;
+
+/// Specifies on which network the code will run.
+enum NetworkId {
+  /// The production network
+  mainnet(magicId: 1),
+
+  /// The test network.
+  testnet(magicId: 0);
+
+  /// The magic protocol number acting as the identifier of the network.
+  final int magicId;
+
+  const NetworkId({required this.magicId});
+}
+
+/// A blockchain slot number.
+extension type SlotBigNum(int value) {
+  /// Serializes the type as cbor.
+  CborValue toCbor() {
```

Comment:
It can also be a getter only and a one-liner.

Reply:
changed to expression, about one liner, lets see the results of other thread where I mentioned toJson() convention